versions.json

```diff
@@ -1,38 +1,30 @@
 [
-  {
-    "version": "6.9.1",
-    "static_tag": "stable"
-  },
-  {
-    "version": "6.9",
-    "static_tag": "mainline"
-  },
-  {
-    "version": "6.8.10",
-    "static_tag": ""
-  },
   {
     "version": "6.6.31",
     "static_tag": "longterm"
   },
   {
-    "version": "6.1.91",
+    "version": "6.10.9",
```

Comment:
For easier navigation  it would be great to keep the ordering of the version from high (at the top) to low (at the bottom). The proposed change mixes this.